pymmmc/datasets/__init__.py

```diff
@@ -1,5 +1,6 @@
 import numpy as np
 import pandas as pd
+from pkg_resources import resource_filename
```

Comment:
Oh! I'd never seen `pkg_resources` before, I had to google it up. From its documentation it looks like its use is discouraged
![image](https://user-images.githubusercontent.com/5230109/207607799-a8dbaf6a-4e63-4006-9955-4648a987411e.png)

I found a [migration guide](https://importlib-resources.readthedocs.io/en/latest/migration.html#migration-guide) that says that you can replace the `resource_filename` with this:

```python
ref = importlib_resources.files('my.package') / 'resource.dat'
with importlib_resources.as_file(ref) as path:
    # Do something with the open path
```

Reply:
Thanks for pointing this out. The changes have been made.